crates/oxc_parser/src/js/statement.rs

```diff
@@ -65,7 +65,7 @@ impl<'a> Parser<'a> {
                                 let src = &self.source_text
                                     [string.span.start as usize + 1..string.span.end as usize - 1];
                                 let directive =
-                                    self.ast.directive(expr.span, (*string).clone(), src);
+                                    self.ast.directive(expr.span, (*string).clone(), Atom::new_inline(src));
```

Comment:
Use `Atom::From`, using `new_inline` will probably break.

Reply:
> thread '<unnamed>' panicked at 'Inline string was too long, max length is `std::mem::size_of::<CompactString>()` bytes', /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/compact_str-0.7.1/src/repr/mod.rs:79:13

Reply:
Got it.